modules/ossm-exposing-service-using-istio-gateway-and-virtualservice.adoc

```diff
@@ -0,0 +1,133 @@
+:_mod-docs-content-type: PROCEDURE
+[id="ossm-exposing-service-using-istio-gateway-and-virtualservice_{context}"]
+= Exposing a service using an Istio gateway and a VirtualService
+:context: ossm-exposing-service-using-istio-gateway-and-virtualservice
+
+This procedure configures a gateway that was deployed using gateway injection to expose a service in the mesh.
+
+.Prerequisites
+
+* You have installed an {istio} gateway using gateway injection.
+
+.Procedure
+
+. Deploy a service in mesh namespace. 
++
+[source,terminal]
+----
+oc create namespace httpbin
+oc label namespace httpbin istio-injection=enabled
+oc apply -n httpbin -f https://raw.githubusercontent.com/openshift-service-mesh/istio/refs/heads/master/samples/httpbin/httpbin.yaml
```

Comment:
We probably need to lock the version down. Unfortunately, I don't think the version of httpbin in the latest `release-1.23` branch is working.

Hi @FilipB, do we know the name of the release branch that will be created for the `openshift-service-mesh/istio` repo for the TP2 release?

Reply:
@eoinfennessy Sorry for delay. It's not yet clear. But I think this version from upstream istio 1.24 release branch should work too - https://raw.githubusercontent.com/istio/istio/refs/heads/release-1.24/samples/httpbin/httpbin.yaml

Reply:
done